Validation/RecoEgamma/plugins/ElectronMcMiniAODSignalValidator.cc

```diff
@@ -15,13 +15,16 @@ typedef edm::Ptr<pat::Electron> PatElectronPtr;
 ElectronMcSignalValidatorMiniAOD::ElectronMcSignalValidatorMiniAOD(const edm::ParameterSet& iConfig) : ElectronDqmAnalyzerBase(iConfig)
 {
     mcTruthCollection_ = consumes<edm::View<reco::GenParticle> >(iConfig.getParameter<edm::InputTag>("mcTruthCollection")); // prunedGenParticles
-    electronToken_ = consumes<pat::ElectronCollection>(iConfig.getParameter<edm::InputTag>("electrons")); // slimmedElectrons
+    electronToken_     = consumes<pat::ElectronCollection>      (iConfig.getParameter<edm::InputTag>("electrons"));         // slimmedElectrons
```

Comment:
hi @archiron - please make the indentation here the same as the rest of the code.